job_definitions/update_index_alias.yml

```diff
@@ -10,7 +10,7 @@
     parameters:
       - string:
           name: ALIAS
-          default: g-cloud
+          default: g-cloud,g-cloud-9
```

Comment:
Do comma-separated aliases in this list get translated into two separate elasticsearch aliases?